twake/frontend/src/app/scenes/Client/MainView/MainView.tsx

```diff
@@ -36,6 +37,13 @@ const MainView: FC = () => {
       {!!channelId && ready && (
         <>
           <AccountStatusComponent />
+          {companyId && (
+            <CompanyBillingBanner
+              companyId={companyId}
+              // TODO ADD URL
+              url=""
```

Comment:
This must not be a props, CompanyBillingBanner can get it from InitService.server_infos....

Reply:
removed